src/commands/utilities.rs

```diff
@@ -21,15 +21,20 @@ pub async fn go(ctx: Context<'_>) -> Result<(), Error> {
 }
 
 /// Links to the bot GitHub repo
-#[poise::command(slash_command, category = "Utilities", discard_spare_arguments)]
+#[poise::command(
+	prefix_command,
+	slash_command,
+	category = "Utilities",
+	discard_spare_arguments
+)]
 pub async fn source(ctx: Context<'_>) -> Result<(), Error> {
 	ctx.say("https://github.com/rust-community-discord/rustbot")
```

Comment:
If you could change this to https://github.com/rust-community-discord/ferrisbot-for-discord, it'd be much appreciated.

Reply:
ran rustfmt and replaced the link.